.github/workflows/aro-hcp-login-check.yml

```diff
@@ -0,0 +1,52 @@
+---
+  name: ARO HCP Continuous Deployment
```

Comment:
just need to update the name to something else so we don't conflict, otherwise lgtm .

Reply:
@stevekuznetsov did you update it?  I'm still seeing 
```
  name: ARO HCP Continuous Deployment